client/packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -47,7 +47,7 @@ export type ColumnKey =
   | 'stockOnHand'
   | 'theirReference';
 
-const getColumnLookup = <T extends RecordWithId>(): Record<
+export const getColumnLookup = <T extends RecordWithId>(): Record<
```

Comment:
I thought to extend column definition from here. Kind of liked the pattern tbh, I appreciate the ability to navigate to code more and more, and just having 'packSize' makes it really hard. But that's just food for thought

Reply:
> Slightly concerned about the non-standard-ness.. and changing only one instance of this established pattern. If we agree on the change, we should add a refactor issue

Agree, and your solution sounds great (I don't think it would require refactor issue ?)

Sounds great: e12336a0c2ee6f3f6f45d6a7af870192d5c390c5